test/Atmos/EDMF/report_mse_bomex.jl

```diff
@@ -12,7 +12,7 @@ include(joinpath(@__DIR__, "compute_mse.jl"))
 data_file = Dataset(joinpath(PyCLES_output_dataset_path, "Bomex.nc"), "r")
 
 #! format: off
-best_mse = Dict()
+best_mse = OrderedDict()
```

Comment:
Same here, I think this should not be in this PR

Reply:
if you run a dry model your MSE table get messed up so I added these changes